File: 1_4/1task.rs
fn main() {
    let N: i32 = 4;
    if N > 26 {
        println!("Incorrect data");
    } else {
        for i in (1..=N).rev() {
            for k in (i..=N).rev() {
                let Str = ('A' as u8 + k as u8 - 1) as char;
                print!("{} ", Str);
            }
            println!();
        }
    }
}

File: 1_4/1task.swift
import Foundation

let N: Int = 4
if N > 26 {
    print("Incorrect data")
} else {
    for i in (1...N).reversed() {
        for k in (i...N).reversed() {
            let Str = Character(UnicodeScalar("A".unicodeScalars.first!.value + UInt32(k - 1))!)
            print(Str, terminator: " ")
        }
        print()
    }
}